sunpy/map/tests/test_plotting.py

```diff
@@ -345,3 +345,25 @@ def test_plot_autoalign(aia171_test_map):
 def test_plot_autoalign_bad_inputs(aia171_test_map):
     with pytest.raises(ValueError):
         aia171_test_map.plot(autoalign='bad')
+
+
+@figure_test
+def test_plot_autoalign_pixel_alignment(aia171_test_map):
```

Comment:
You will need to add this hash to the two figure hash files. You can get the hash and test name from the failed CircleCi runs.